docker/dev/docker-compose.yml

```diff
@@ -1,75 +1,109 @@
-version: '3.3'
+version: "3.3"
+
+networks:
+  # communication to web and clients
+  lemmyexternalproxy:
+  # communication between lemmy services
+  lemmyinternal:
+    driver: bridge
+    internal: true
 
 services:
-  nginx:
+  lemmy-proxy:
     image: nginx:1-alpine
+    container_name: lemmy-proxy
+    networks:
+      - lemmyinternal
+      - lemmyexternalproxy
     ports:
-      - "1236:1236"
+    # actual and only port facing any connection from outside 
+      - "1239:1236"
```

Comment:
Dont change the external port, thats confusing.

Reply:
OK, i find it less confusing honestly. Also i wanted a different port from prod. Let's say you run a dev on the same server where a non-dev already runs? This is very convenient. But we can switch it back of course.